com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/SampleVirtualTextureNode.cs

```diff
@@ -426,62 +426,68 @@ public void GenerateNodeFunction(FunctionRegistry registry, GenerationMode gener
 
         public void GenerateNodeCode(ShaderStringBuilder sb, GenerationMode generationMode)
         {
+            bool success = false;
             if (IsSlotConnected(VirtualTextureInputId))
             {
                 var vtProperty = GetSlotProperty(VirtualTextureInputId) as VirtualTextureShaderProperty;
-                int layerCount = vtProperty.value.layers.Count;
-
-                var layerOutputVariables = new List<string>();
-                for (int i = 0; i < layerCount; i++)
+                if (vtProperty != null)
```

Comment:
Github diff is getting confused by the added indent from the new { } block added by this if()..  there are no changes in the block below other than adding success= true.